src/json/converter/csv/CSVReader.java

```diff
@@ -121,7 +122,7 @@ public void read(boolean readHeader){
 
 		try {
 
-			_reader = new BufferedReader(new InputStreamReader(_instream));
+			_reader = new BufferedReader(new InputStreamReader(_instream);
```

Comment:
This change results in a compilation error.

Reply:
updated